README.md

```diff
@@ -34,7 +34,9 @@ The Current constitution was written in 2017. (Which tracking starts from, all p
   1.0.4.0 Added a section for the history of the constitution to be recorded, I'm expecting it to reference these change numbers.  
   1.0.4.1 Formalised "Policies" as an alternative to regulations, this is simply so the committee can have choice of wording.  
   1.0.5.0 Addition of the section on subcommittees, and the addition of the "Custodes" subcommittee, along with a template for new ongoing subcommittees.  
-
+  1.0.6.0 Addition of the Marketing Officer Position to the executive committee section
+  
+  1.0.6.1 Renamed, and merged the Womens/Mens Representative position into the Gender Equity represenatative; added the title of Commander-of-Chief of the Cyber, and   Space force to the President, minor grammatical changes; added a clause to strip honorary membership 
```

Comment:
Please fix spelling errors:

- Women's
- Men's
- Representative
- Commander-in-Chief

Please fix various spacing errors.